AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -353,6 +352,10 @@ - (MSALAccount *)retrieveAccount {
   }
 }
 
+- (NSString *)getAccountId {
```

Comment:
We should call it `retrieveAccountId:` to align with `(MSALAccount *)retrieveAccountWithAccountId:(NSString *)homeAccountId` API.